pkg/controller/wip/controller.go

```diff
@@ -382,16 +387,30 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 	glog.V(4).Infof("Creating client for Broker %v, URL: %v", broker.Name, broker.Spec.URL)
 	brokerClient := c.brokerClientCreateFunc(broker.Name, broker.Spec.URL, username, password)
 
-	// TODO: parameters
+	parameters, err := unmarshalParameters(instance.Spec.Parameters.Raw)
```

Comment:
well, since I was asked to pull it into a method, I moved it there. Otherwise, there's really no point abstracting away a call to yaml.unmarshal. Also, since the method returns a blank parameters, don't need to initialize here, ok?